packages/runtime/lib/worker/app.js

```diff
@@ -185,6 +186,15 @@ class PlatformaticApp extends EventEmitter {
     }))
     process.exit(1)
   }
+
+  #wrapStackable (stackable) {
+    for (const method of Object.keys(defaultStackable)) {
+      if (!stackable[method]) {
+        stackable[method] = defaultStackable[method]
+      }
+    }
+    return stackable
```

Comment:
let's not modify the current object as it can have odd side effects.
How about you do:

```js
const newStackable = Object.create(stackable)
```

and then add the missing methods instead?

Reply:
done. I did in a bit different way, because `Object.create(stackable)` would prevent from using private methods in a original class instance.